code/client/makepkginfo

```diff
@@ -385,6 +385,15 @@ def check_mode(option, opt, value, parser):
                                                             (opt, mode))
     setattr(parser.values, option.dest, value)
 
+def has_valid_install_critieria(catinfo):
```

Comment:
What happens if you import a configuration profile? I imagine we'll get a bogus warning.